### ISSUE TITLE
Is preset-import-file working properly?

### ISSUE THREAD

**Issue by Marcosaurios** [75lb/handbrake-js#37]
Hi! 
I'm trying to compress videos by reading an exported custom preset created on HandBrake GUI.

The problem is that I'm not being able to compress the video using this custom preset. The ```sameWidthSameFps.json``` file is on the same directory where```app.js``` is running under node.

```
hbjs.spawn({ 
        'preset-import-file': "./sameWidthSameFps.json",
        'preset': 'sameWidthSameFps',
        input: filepath,
        output: outputEnd
    })
    .on('error', err => {
        console.log(err);
    })
```

But if I leave ```input``` and ```output``` flags alone it works fine. Anyone knows anything? Thanks!

**Comment by 75lb** [comment]
Hi, could you post the `sameWidthSameFps.json` file please to help me reproduce this

**Comment by Marcosaurios** [comment]
Thank you!
Here is the file. GitHub don't support to upload ```.json``` files...
https://drive.google.com/open?id=1s8Fm3vPMD7giIvUOVLZiWs8WciY5k0un

**Comment by asharafshahi** [comment]
I can confirm the same issue is happening for me.  Anytime I specify 'preset-import-file' as an option the spawn simply exists with no error.  Has anyone figured out a work-around for this?

**Comment by Marcosaurios** [comment]
@asharafshahi nope, I didn't find any way to make it work...

**Comment by 75lb** [comment]
i have reproduced this issue, it's related to #16 and #25.. internally, handbrake-js spawns a HandbrakeCLI process.. however, some HandbrakeCLI command line args require you to use `--option=value` notation while others fail with that notation, requiring you to use `--option value` notation.. it's a random inconsistency which has caused a few bugs - will fix ASAP.

**Comment by asharafshahi** [comment]
Wow ok, that makes sense and thanks very much for looking into it!

**Comment by wilko2205** [comment]
Is there any workaround in the meantime? Looks like --preset-import-gui has the same issue

**Comment by 75lb** [comment]
Fixed and released in v4.0.3.

I've [added exclusions](https://github.com/75lb/handbrake-js/blob/master/lib/Handbrake.js#L78) for `--preset-import-file`, `--preset-import-gui` and `--subtitle-burned` - these options will no longer be passed to the underlying `HandbrakeCLI` tool using `--options=value` notation. 

I've also improved the error handling to prevent handbrake-js failing silently if this issue arises again, instead you'll get an error message like this: 

```
Other: HandbrakeCLI reported "unknown option", please report this issue: https://github.com/75lb/handbrake-js/issues/new
    at Handbrake._emitOutput (/Users/lloyd/Documents/75lb/handbrake-js/lib/Handbrake.js:217:19)
    at Socket.emit (events.js:210:5)
    at addChunk (_stream_readable.js:308:12)
    at readableAddChunk (_stream_readable.js:289:11)
    at Socket.Readable.push (_stream_readable.js:223:10)
    at Pipe.onStreamRead (internal/stream_base_commons.js:182:23) {
  name: 'Other',
  message: 'HandbrakeCLI reported "unknown option", please report this issue: https://github.com/75lb/handbrake-js/issues/new',
  output: '[22:41:58] hb_init: starting libhb thread\n' +
    '[22:41:58] thread 70000578c000 started ("libhb")\n' +
    'unknown option (--preset-import-gui=./tmp/issue-37.json)\n',
  options: {
    'preset-import-gui': './tmp/issue-37.json',
    preset: 'sameWidthSameFps',
    input: './tmp/test.mp4',
    output: './tmp/test.mkv'
  }
}
```
